Aufgabe3/gta_v3/models/geotag-store.js

```diff
@@ -24,9 +25,69 @@
  * - Keyword matching should include partial matches from name or hashtag fields. 
  */
 class InMemoryGeoTagStore{
+    #geoTags = [];
+
+    addGeoTag(geotag) {
+        this.#geoTags.push(geotag);
+    }
+
+    removeGeoTag(name) {
```

Comment:
Falls name als der Name von der Position bei GeoTag gemeint ist (also zB. "HSKA Campus, Gebäude B"), fehlt dann nicht ein this.#geoTags[i].getName in der if unten?

Reply:
Ja logisch, hab es da vergessen. Sowas fällt beim testen normal super schnell auf, aber kann es nicht testen :D

Reply:
Ja :D